Scripts/methods.js

```diff
@@ -1,15 +1,54 @@
 'use strict';
+var myProjects = [];
 
 PortfolioData.prototype.createImage = function(i) {
   var imageEl = document.createElement('img');
   return imageEl;
 };
 
 PortfolioData.prototype.toHtml = function() {
-  var $newProject = $('article.about-me').clone().removeClass('about-me');
-  $newProject.find('h3').text(this.projName);
-  $newProject.find('.project-image').attr('src', this.icon);
-  $newProject.find('.description').text(this.description);
-  $newProject.find('.date').text(this.date);
-  return $newProject;
+  var source = $('#project-template').text();
+  var templateRender = Handlebars.compile(source);
+
+  return (templateRender(this));
+};
+
+projects.forEach(function(projectObject){
+  myProjects.push(new PortfolioData(projectObject));
+});
+
+function showHome(){
```

Comment:
IF we get some free time(hahahaa i know. Free time, what is that?), I'd love to sit and refactor these functions together! They are really awesome, but we want as little on the global space as possible.